assets/js/wcsatt-single-add-to-cart.js

```diff
@@ -31,7 +31,7 @@
 				var $scheme_option = $( this ),
 					scheme_data    = $( this ).find( 'input' ).data( 'custom_data' );
 
-				bundle.satt_schemes.push( { el: $scheme_option, data: scheme_data } );
+				bundle.satt_schemes.push( { el: $scheme_option, data: scheme_data, price_html_o: $scheme_option.find( '.subscription-price' ).html(), details_html: $( '<div>' ).html( $scheme_option.find( '.subscription-details' ) ).html() } );
```

Comment:
`_o` stands for "original". The original/initial price html in the subscription options list is replaced by a "live" version, which is calculated based on the chosen options/configuration of a bundle/composite.

e.g. "From $X" could become "$Y" while/after configuring a bundle.